### PR TITLE
WT-7133 Fix bug in stat collection when target pages are reduced due to high HS cache pressure

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

File: src/evict/evict_lru.c [hunk 1]
```diff
@@ -1741,9 +1741,11 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
      */
     if (!WT_IS_HS(btree->dhandle) && __wt_cache_hs_dirty(session)) {
         /* If target pages are less than 10, keep it like that. */
-        target_pages = target_pages < 10 ? target_pages : target_pages / 10;
-        WT_STAT_CONN_INCR(session, cache_eviction_target_page_reduced);
-        WT_STAT_DATA_INCR(session, cache_eviction_target_page_reduced);
+        if (target_pages >= 10) {
+            target_pages = target_pages / 10;
+            WT_STAT_CONN_INCR(session, cache_eviction_target_page_reduced);
```
Comment (hbokhari) [inline src/evict/evict_lru.c:1746]: There is a new macro `WT_STAT_CONN_DATA_INCR` that can increase both conn and data stats in one go. Can you try if we can use that macro here?

Comment (mm-ng) [reply src/evict/evict_lru.c:1746]: Yes, I believe that can be used here, I will make that change.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by hbokhari — APPROVED** [review]
LGTM!

**Pull request merged**